### PR TITLE
Support specifying load balancer ip range and ingress lb ip

### PR DESCRIPTION
**Pull request by klausenbusk**
It is part of the network configuration so should be handled here.

---

It has been cleared with @jakobhviid.

### CODE REVIEW

**Comment by klausenbusk** [comment]
I have tested the `helm install` command and the values seem to be passed correctly.

File: templates/boot.sh [hunk 1]
```diff
@@ -46,7 +46,11 @@ case $NODE_TYPE in
         for FILE in /root/helm-charts/*; do
             release=$(echo $FILE | cut -f4 -d/ | cut -f1 -d#)
             namespace=$(echo $FILE | cut -f4 -d/ | cut -f2 -d#)
-            helm install --create-namespace -n $namespace $release $FILE
+            helm install --create-namespace -n $namespace \
+              --set yggdrasil.loadbalancer.ipRangeStart=$$LB_IP_RANGE_START \
+              --set yggdrasil.loadbalancer.ipRangeStop=$$LB_IP_RANGE_STOP \
+              --set yggdrasil.ingress.loadBalancerIP=$$INGRESS_LB_IP_ADDRESS \
+              $release $FILE
```
Comment (skovbakke) [inline templates/boot.sh:53]: I see that this works. However, currently the script was made to support installing any helm chart, and parts of the script were made to support this.  
If we decide to use these cmd-line options (which set values specific to yggdrasil) we might as well move the helm chart value into the config file as well. The makefile will be simpler then. @jacobcrawford what do you think?

**Review by skovbakke — CHANGES_REQUESTED** [review]

File: templates/boot.sh (hunk 1 above)
Comment (klausenbusk) [reply templates/boot.sh:53]: https://github.com/distributed-technologies/mukube-configurator/pull/33 is a more generic approach, but IMO it make sense to be explicit about some of the options (lb, proxies etc.).

Comment (jacobcrawford) [reply templates/boot.sh:53]: I dont think i see why we need to handle this configuration here and not in yggdrasil?

**Review by skovbakke — APPROVED** [review]
LGTM. I have created a task to refactor the configurator based on the talk we had today

**Pull request merged**